info.json

```diff
@@ -1,20 +1,19 @@
 {
 	"name": "laserfence",
-	"title": "⚡ C&C Laserfence",
-	"description": "⚡Use Laser Fence technology that was developed during the Tiberium Wars to fend off biters.",
+	"title": "Laser Fence",
+	"description": "Fend off biters with Laser Fence technology from the Command and Conquer universe that was developed during the Tiberium Wars.",
 
-	"author": "randomflyingtaco, Sacredanarchyc",
-    "contact": "https://github.com/randomflyingtaco/laserfence",
-    "homepage": "https://discord.gg/freXKz5",
+	"author": "randomflyingtaco and Sacredanarchy",
+	"contact": "https://github.com/randomflyingtaco/laserfence",
+	"homepage": "https://discord.gg/freXKz5",
```

Comment:
if you use tab and not space you will have some problem, don't ask me why, i have no idea why it's make mage crash

Reply:
Tabs are fine to indent in info.json maybe you are thinking of changelog.txt which require 2 spaces for indenting?